auth_backend/http.py

```diff
@@ -0,0 +1,7 @@
+HTTP_200_OK = 200
+HTTP_201_CREATED = 201
+HTTP_204_NO_CONTENT = 204
+HTTP_404_NOT_FOUND = 404
+HTTP_409_CONFLICT = 409
+HTTP_422_UNPROCESSABLE_ENTITY = 422
+HTTP_500_INTERNAL_SERVER_ERROR = 500
```

Comment:
yeah, i saw that, but they dont include the error code in their naming, and it is just a list of constants